eng/build.sh

```diff
@@ -34,15 +36,38 @@ while (($# > 0)); do
       arguments="$arguments --rebuild"
       shift 1
       ;;
-      --)
+     -pack)
+      defaultargs="--pack"
+      shift 1
+      ;;
+     -restore)
+      defaultargs="--restore"
+      shift 1
+      ;;
+     -publish)
+      defaultargs="--build /p:PublishPackages=true"
       shift 1
       ;;
       *)
-      extraargs="$extraargs $1"
+      ea=$1
+
+      if [[ $checkedPossibleDirectoryToBuild == false ]]; then
```

Comment:
We don't need this extra logic in the standard repo. It is just a simplification for corefx.